examples/opensora_hpcai/opensora/schedulers/iddpm/gaussian_diffusion.py

```diff
@@ -299,7 +295,7 @@ def p_sample(
         """
         if frames_mask is not None:
             if frames_mask.shape[0] != x.shape[0]:
-                frames_mask = frames_mask.reshape(1, -1).repeat(2, axis=0)  # HACK
+                frames_mask = repeat_interleave(frames_mask.reshape(1, -1), 2, 0)  # HACK
```

Comment:
Leave it as it is since `frames_mask` is a numpy array.

Reply:
fixed